src/pages/en/guides/deploy/github.md

```diff
@@ -25,45 +25,60 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
     name: Github Pages Astro CI
 
     on:
-      # Trigger the workflow every time you push to the `main` branch
-      # Using a different branch name? Replace `main` with your branch’s name
+     # Trigger the workflow every time you push to the `main` branch
       push:
-        branches: [main]
+        branches: [ main ]
       # Allows you to run this workflow manually from the Actions tab on GitHub.
       workflow_dispatch:
+      
+      # Allow this job to push changes to your repository
```

Comment:
Minor rephrasing. Part of the update to the new workflow is we no longer require a token that can push to the repo directly:

```suggestion
      # Allow this job to clone the repo and create a page deployment
```